addon/evaluate-conditions.js

```diff
@@ -1,4 +1,11 @@
+/**
+ * @module evaluate-conditions
```

Comment:
I'd think unless our README documents consuming this file (which it doesn't), then we can consider this a private API change and thus if a consumer were consuming it, the fallout is on them.